src/btree/bt_walk.c

```diff
@@ -13,14 +13,14 @@
  *	Move to the next/previous page in the tree.
  */
 int
-__wt_tree_walk(WT_SESSION_IMPL *session, WT_REF **refp, uint32_t flags)
+__wt_tree_walk(WT_SESSION_IMPL *session,
+    WT_REF **refp, uint64_t *refcntp, uint32_t flags)
```

Comment:
@michaelcahill, I'd rename `refcntp` to be `walkcntp`, or something more descriptive.